### DISCUSSION TITLE
[Unstable] Couldn't generate beacon in End level

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14594] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Game couldn't generate beacon in End level on Unstable
Works just fine in Stable

Tried with and without mods

![image](https://github.com/user-attachments/assets/d0d15c92-c0e1-4600-a2cf-f0bb73b83bdc)
![image](https://github.com/user-attachments/assets/0fb123d5-b43c-4c61-8dc4-93ae77f4ac16)

[Save 5 - Copy.zip](https://github.com/user-attachments/files/16923043/Save.5.-.Copy.zip)

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, the fix is coming in the next Unstable build.
